src/utils/systemInfo.js

```diff
@@ -0,0 +1,44 @@
+const { loadavg, cpus, networkInterfaces, uptime, freemem, totalmem } = require('os');
+
+const byteToMB = (byte) => {
+	if (byte <= 0) {
+		return byte;
+	}
+	return byte / 1024 / 1024;
+};
+
+const pid = () => process.pid;
```

Comment:
Why wrap this in a function? The pid will not change, so we can fetch it once.

Reply:
true